doc/source/ray-overview/index.rst

```diff
@@ -0,0 +1,160 @@
+.. _gentle-intro:
+
+============================
+A Gentle Introduction to Ray
+============================
+
+.. include:: basics.rst
+
+This tutorial will provide a whirlwind tour of all that Ray has to offer.
+
+First, install Ray with: ``pip install ray``, now we can execute some Python in parallel.
+
+Parallelizing Python Functions with Ray Tasks
+=============================================
+
+First, import ray and ``init`` the Ray service.
+Then decorate your function with ``@ray.remote`` to declare that you want to run this function 
+remotely. Lastly, call that function with the ``.remote`` method call, this yields futures that you can then 
```

Comment:
```suggestion
remotely. Lastly, call that function with ``.remote()`` instead of calling it normally. This remote call yields a future, or ``ObjectID`` that you can then
```